scripts/qiita-recover-jobs

```diff
@@ -16,7 +16,7 @@ from time import sleep
 SLEEP_TIME = 2
 
 
-def _submit_jobs(jids_to_recover, recover_type):
+def Qsubmit_jobs(jids_to_recover, recover_type):
```

Comment:
If we are fixing Click, do we really need these changes? Either way, this script does _not_ use click so should not be affected, right? If that's the case, please revert.

Reply:
I don't think it needs it. Changing functions and methods starting with '_' to starting with 'Q' didn't change anything. I'll revert it.